src/main/java/com/chat/yourway/exception/ServiceException.java

```diff
@@ -0,0 +1,17 @@
+package com.chat.yourway.exception;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+
+@Getter
+@RequiredArgsConstructor
+public class ServiceException extends RuntimeException {
```

Comment:
You should make clear and understandable exceptions. When someone will open your code (who havent checked it previously), he wouldn`t understand what exactly should happen to get this exception. It could be everything